nav-logged-in.php

```diff
@@ -0,0 +1,8 @@
+<?php  if($_SESSION['login']) : ?>
```

Comment:
Same thing here, you can directly exit out of the template if the user is not logged-in.
```suggestion
<?php 
if ( ! isset( $_SESSION['login'] ) ) {
    return;
}
?>
```

Reply:
fixed